nestkernel/recording_backend_sionlib.h

```diff
@@ -75,12 +79,39 @@ filename is composed.
 Data format
 +++++++++++
 
-The container files contain the data in a custom binary format. The
-recorder property ``label`` and all other recorder-specific metadata
-is stored with each recording device. To retrieve the data after the
-simulation, the `nestio reader API` in Python has to be used. SIONlib
-container files are sparse files: They only use up the space on disk
-which is actually filled with data.
+In contrast to other recording backends, the ``sionlib`` backend
+writes the data from all recorders using it to a single container
+file(s). The file(s) contain the data in a custom binary format, which
+is composed of a series of blocks in the following order:
+
+* the *body block* contains the actual data records; the layout of an
+  individual record depends on the type of the device and is described
+  by a corresponding entry in the *device info block*
+  
+* the *file info block* keeps the file's metadata like version
+  information and such.
+
+* the *device info block* stores the properties and a data layout
+  description for each device that uses the ``sionlib`` backend.
+
+* the *tail block* contains pointers to the *file info block*
```

Comment:
Two of the four bullet points here end with a period. Also, as they are all full sentences, they should start with a capital letter. Periods are not absolutely essential, but it should be consistent.

Reply:
Fixed by bcaae82f9.